examples/fedopt_example/metrics.py

```diff
@@ -107,7 +107,8 @@ def summarize(self) -> str:
         log_string = f"{log_string}\naverage_f1:{str(sum_f1/n_topics)}"
         return log_string
 
-    def update_performance(self, predictions: np.ndarray, labels: np.ndarray) -> None:
+    def update_performance(self, predictions: torch.Tensor, labels: torch.Tensor) -> None:
```

Comment:
For some reason, this started throwing pre-commit issue (correctly asserting this function is being passed torch tensor but the types of the argument were numpy ndarrays) even though it had previously passed and there was nothing changed in my code that would cause it to fail all of the sudden. I noticed this came up in Claire's PR's pre-commit checks on the github server too. I suspect since we dont have versions pinned, when we setup the environment, its possible to have inconsistent versions, leading to potentially inconsistent pre-commit behaviour. I could be wrong though, if you have any ideas let me know.

Reply:
Yeah I think it has to do with the version not being pinned. It's shown up for my PR and Yuchong's as well. However, it's hard to argue that it doesn't make sense. Yuchong's fix is slightly more general. Maybe we can just adopt that one?